src/EnergyPlus/SizingManager.cc

```diff
@@ -3011,9 +3011,9 @@ void GetZoneSizingInput(EnergyPlusData &state)
                 if (state.dataIPShortCut->lNumericFieldBlanks(3)) {
                     state.dataSize->ZoneSizingInput(ZoneSizIndex).HeatDesTemp = 0.0;
                 } else if (state.dataSize->ZoneSizingInput(ZoneSizIndex).ZnHeatDgnSAMethod == SupplyAirTemperature) {
-                    ReportTemperatureInputError(state, cCurrentModuleObject, 1, lowTempLimit, false, ErrorsFound);
+                    ReportTemperatureInputError(state, cCurrentModuleObject, 3, lowTempLimit, false, ErrorsFound);
                     ReportTemperatureInputError(
-                        state, cCurrentModuleObject, 1, state.dataSize->ZoneSizingInput(ZoneSizIndex).CoolDesTemp, true, ErrorsFound);
+                        state, cCurrentModuleObject, 3, state.dataSize->ZoneSizingInput(ZoneSizIndex).CoolDesTemp, true, ErrorsFound);
                     state.dataSize->ZoneSizingInput(ZoneSizIndex).HeatDesTemp = state.dataIPShortCut->rNumericArgs(3);
```

Comment:
This is also been tested to working using a defect file that contains a -20C supply temperature in a Sizing:Zone object. 

In a quick summary it is working as expected and fixed the problem well---and actually in this defect file case, even though the Sizing:Zone has an input value of -20C for supply temperature, it should not cause a warning in the defect file as the defect file as the object actually specifies a "temperature difference" method---and this should also add to the point when it is a bug before. After the fix the problem is fixed; and no warning for the defect file on this zone (because it uses the temperature difference method): 
![image](https://user-images.githubusercontent.com/64040111/130521080-282f7d83-0c9a-456c-baaf-1bc4ed954a1b.png)

A little bit complication about this test file is that with the difference method, the user specified a 11.11C temperature difference in Sizing:Zone and the zone thermostat is set to 2C---this means that the supply is around -9.11. I original thought that this is the reason why the warning is flagged.--but actually it should not be. According to the code's original intention, it should only flag the below zero supply temperature with `SupplyAirTemperature` method. Therefore from this point it is also a bug. 

I think the new fix nice fixed this and it is now clear that the warning is given only when a below-zero supply air temperature is inputted together with the "SupplyAirTemperature" method. If a user selects the "temperature difference", then even though a negative supply air temperature is inferred from the inputs, it would not (and also probably should not) give a warning.